test/unit/keystore/v4.spec.ts

```diff
@@ -32,55 +53,37 @@ describe("Eth1ICGKeystore", () => {
 
 
     it("should create keystore", async () => {
```

Comment:
This test is probably useless. It would probably make sense to have something like V4Keystore.fromJSON(JSON.stringify(example)) in beforeach. And this first test should test creating of new keystore.

It would be a lot more cleaner